expr/expr.go

```diff
@@ -271,10 +278,17 @@ func (e expr) consumeKwarg(key string, optArgs []Arg) error {
 		}
 		*v.val = got.str
 	case ArgBool:
-		if got.etype != etBool {
-			return ErrBadKwarg{key, exp, got.etype}
+		if got.etype == etBool {
+			*v.val = got.bool
+			break
+		}
+		if got.etype == etString {
+			if val, wasBool := strToBool(got.str); wasBool {
```

Comment:
how about we make strToBool a method on `expr` ? we just comment that it's only defined for etString types.

Reply:
We use the same logic in parse.go on the string `e`.